### PR TITLE
FWF-3945: [Bugfix] Resolved pagination issues in submission,draft,client tables

### PR DESCRIPTION
**Pull request by Bonymol-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3945
Issue Type: BUG/ FEATURE

# Changes
* Fixed pagination issues in submission, draft, client tables

# Screenshots (if applicable)

# Notes

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

**Review by fahad-aot — APPROVED** [review]

File: forms-flow-web/src/components/Application/ApplicationTable.js [hunk 1]
```diff
@@ -314,47 +312,19 @@ const ApplicationTable = () => {
...
-                linkClass="page-link"
-                onChange={handlePageChange}
-              />
-            </div>
-          </div>
-        ) : null}
+          <table className="table">
+            <tfoot>
+             <TableFooter
+            limit={limit}
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Application/ApplicationTable.js:318]: if there is no data need to render some message right @arun-s-aot

File: forms-flow-web/src/components/Form/constants/ClientTable.js [hunk 2]
```diff
@@ -289,54 +286,20 @@ function ClientTable() {
...
-          </div>
-        </div>
-      ) : (
-        ""
-      )}
+        <table className="table">
+          <tfoot>
+          <TableFooter
+            limit={limit}
+            activePage={pageNo}
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/constants/ClientTable.js:293]: in every case @arun-s-aot  need to show some message if there is no data in that table ?

Comment (arun-s-aot) [reply forms-flow-web/src/components/Form/constants/ClientTable.js:293]: Yes. Fahad's PR will solve that. @shuhaib-aot .

File: forms-flow-web/src/components/Application/ApplicationTable.js (hunk 1 above)
Comment (arun-s-aot) [reply forms-flow-web/src/components/Application/ApplicationTable.js:318]: It will be done by fahad in his pr

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**